src/Framework/test/SharedFxTests.cs

```diff
@@ -114,19 +114,33 @@ public void ItContainsValidDepsJson()
             Assert.NotNull(depsFile["compilationOptions"]);
             Assert.Empty(depsFile["compilationOptions"]);
             Assert.NotEmpty(depsFile["runtimes"][_expectedRid]);
-            Assert.All(depsFile["libraries"], item =>
```

Comment:
Can you undo the changes to this test? The test in master is correct and should be left untouched. The changes I made in 2.x were to backport a portion of this test.